cmux-tui/spec/commands.md

```
@@ -1373,15 +1463,29 @@ CLI mapping: verb `process-info`; flags `--surface <id>`; plain stdout prints `p
+although browser color application is a no-op.
 
 Params:
 
 | Name | JSON type | Required/default | Constraints |
 | --- | --- | --- | --- |
 | `fg` | `ColorHex` | default null | Foreground color |
 | `bg` | `ColorHex` | default null | Background color |
+| `cursor` | `ColorHex` | default null | Cursor color |
+| `selection_bg` | `ColorHex` | default null | Selection background |
+| `selection_fg` | `ColorHex` | default null | Selection foreground |
+| `cursor_style` | `"block"|"underline"|"bar"` | default null | Cursor shape |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)